src/components/RichTextEditor/TextStyleDropdown/DropdownTrigger.tsx

```diff
@@ -1,45 +1,86 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { IconCaretDown } from '@foundation/Icon';
-import { getParent, getPreventDefaultHandler, usePlateEditorState } from '@udecode/plate';
-import { merge } from '@utilities/merge';
-import React from 'react';
-import { TextStyles, textStyleTitles } from '../utils/getTextStyles';
+import { IconCaretDown } from "@foundation/Icon";
+import {
+    PlateEditor,
+    getNodes,
+    getPreventDefaultHandler,
+    isElement,
+    isType,
+    usePlateEditorState,
+} from "@udecode/plate";
+import { merge } from "@utilities/merge";
+import React from "react";
+import { TextStyles, textStyleTitles } from "../utils/getTextStyles";
 
 type DropdownTriggerProps = {
     editorId?: string;
     open: boolean;
 };
+const DEFAULT_TEXTSTYLE_VALUE = "Mixed";
+
+enum ListType {
+    UL = "ul",
+    OL = "ol",
+    CHECKLIST_ITEM = "checkbox_item",
+}
+
+const listTitle: Record<ListType, string> = {
+    [ListType.UL]: "Bullet List",
+    [ListType.OL]: "List",
+    [ListType.CHECKLIST_ITEM]: "Checklist",
+};
+
+type AvailableTextStyles = ListType & TextStyles;
+
+const availableTextStyleTitles: Record<AvailableTextStyles, string> = { ...listTitle, ...textStyleTitles };
```

Comment:
you could rename this to `TEXT_STYLE_TITLES` to keep it shorter and uppercase for a file constant